cmd/gpu_plugin/README.md

```diff
@@ -16,7 +16,7 @@ Table of Contents
 ## Introduction
 
 Intel GPU plugin facilitates Kubernetes workload offloading by providing access to
-Intel discrete (Xe) and integrated GPU HW device files.
+Intel discrete (including Intel® Data Center GPU Flex Series) and integrated GPU HW device files.
```

Comment:
Maybe "discrete (...branding...) and integrated Intel GPU device files"?

(move "Intel" to front of "GPU" like in main README.)